tests/mir-opt/dataflow-const-prop/enum.rs

```diff
@@ -50,7 +90,11 @@ fn mutate_discriminant() -> u8 {
             // This assignment overwrites the niche in which the discriminant is stored.
             place!(Field(Field(Variant(x, 1), 0), 0)) = 0_usize;
             // So we cannot know the value of this discriminant.
+
+            // CHECK: [[a:_.*]] = discriminant(_{{[0-9]*}});
```

Comment:
```suggestion
            // CHECK: [[a:_.*]] = discriminant({{_.*}});
```
And likewise everywhere.